### DISCUSSION TITLE
Still in developement?

### DISCUSSION THREAD

**Discussion by Ubimo** [rand256/valetudo#469] [Q&A] [answered]
Hello
I see the last release is from 5 months ago. Is this project still in developement, or EOL?

**Comment by rand256** [comment]
What kind of "development" exactly would you like to see on S5x and gen1 devices?

**Comment by Ubimo** [comment]
Original Valetudo is still beeing developed, see: https://github.com/Hypfer/Valetudo
Would you please include the original Valetudo updates in your Valetudo RE?

**Comment by Ubimo** [comment, marked as answer]
So, EOL?

**Reply by rand256** [reply]
Yeah. While I still may add something I could feel useful for 1-2 gen devices, I definitely will not add support for other new devices, cause, you know, I don't own those. Moreover, I could also admit I was quite disappointed in roborock these years, which doesn't make things better as well.

Regarding your suggestion to update RE to current vanilla state it's like asking to throw away current RE and rewrite all its features from scratch again upon new vanilla code base. It is definitely possible, but why? So, sure, better take it as EOL.

**Reply by Braintoe** [reply]
@rand256 Yes, I fully understand that - the original Valetudo has changed a lot since you started the fork.
Well then, thank you very, very much for doing this development. My cleaner will definitely stay on RE as long as it works since RE is the only variant providing a localised interface.

If I may, I would like take your localisation work as a basis for an attempt to do a localised Valetudo GUI when the time will be ready for a new cleaner.

When it comes to listening to the input from users, Hypfer definitely could learn a lot from you! And I firmly believe Valetudo would lack some of the features it has now if you would not have released RE ;-)
